webClient/src/app/shared/editor-control/editor-control.service.ts

```diff
@@ -628,6 +705,54 @@ export class EditorControlService implements ZLUX.IEditor, ZLUX.IEditorMultiBuff
     return _observable;
   }
 
+  
+  saveDatasetHandler(context?: ProjectContext, results?: any): Observable<void> {
+    const _openFile = this.openFileList.getValue();
+    let _activeFile: ProjectContext;
+    let _observer: Observer<void>;
+    let _observable: Observable<void>;
+    let sessionID: number;
```

Comment:
sessionID is never used

Reply:
Correct.  This was inserted for an earlier solution which is now abandoned.  I'll remove the declaration.